### ISSUE TITLE
blockstack-server: command not found

### ISSUE THREAD

**Issue by JustinDrake** [blockstack/blockstack-core#247]
I've successfully run `pip install blockstack` on Ubuntu but now I get `command not found`:

```
root@api:~/backend# blockstack-server start
blockstack-server: command not found
```

Did I miss something?

**Comment by pandu-rao** [comment]
I'm not sure if there is a blockstack-server anymore. I remember hearing that blockstack uses bitcoind directly.

I happen to have bitcoind installed and the entire blockchain available on my workstation.

Maybe someone else can confirm if this is true.

In the meanwhile, you may want to install bitcoind and ensure that the full blockchain is available on your workstation.

**Comment by tthsn** [comment]
Fix: pip install blockstack-server

**Comment by tthsn** [comment]
Don't know why github page of blockstack-core says that "pip install blockstack".

**Comment by muneeb-ali** [comment]
The 0.13.x release used blockstack-server. For 0.14 we'll rename to blockstack-core. 

0.14 is not active yet, so yes, for now, you need to do:

> pip install blockstack-server 

And we definitely need to update the docs. Feel free to send a pull request!

**Comment by JustinDrake** [comment]
I have Blockstack cli version 0.14.0.7 (installed with `pip install blockstack`), and I still cannot get `blockstack-server` to run. What am I doing wrong?

**Comment by muneeb-ali** [comment]
what's the error you're getting when you run blockstack-server?

**Comment by JustinDrake** [comment]
Inside virtualenv:

```
(blockstack) Justins-MacBook-Pro:blockstack justin$ blockstack-server
-bash: blockstack-server: command not found
```

**Comment by jcnelson** [comment]
You have to install the blockstack core package as well (with `pip install blockstack-core`).

**Comment by muneeb-ali** [comment]
@JustinDrake this has been fixed in the v0.14.1. Closing the issue.
